dataland-frontend/src/components/general/CompanyDataTable.vue

```diff
@@ -71,7 +71,9 @@ import DataTable from "primevue/datatable";
 import Column from "primevue/column";
 import DetailsCompanyDataTable from "@/components/general/DetailsCompanyDataTable.vue";
 import { detailsCompanyDataTableColumnHeaders } from "@/components/resources/frameworkDataSearch/lksg/DataModelsTranslations";
-import { ReportingPeriodOfDataSetWithId } from "@/utils/DataTableDisplay";
+import { expandRowGroupOnHeaderClick, ReportingPeriodOfDataSetWithId } from "@/utils/DataTableDisplay";
+
+type SfdrKpiObject = { [index: string]: string | object; subAreaKey: string; kpiKey: string };
```

Comment:
The types we have in KpiDataObjects.ts should be used in `SfdrPanel.vue`

Reply:
Looking at this...

Reply:
KpiDataObject does not contain the `subAreaKey` required by SFDR. 
I'm not 100 % sure that all keys/properties of the KpiDataObject align with SFDR requirements.
Still looking ...

Reply:
Ok I re-checked this. A couple of things are a little confusing reading the code, which made me think that this was possible to do:

1.) The component `CompanyDataTable` is actually a table-view-component which is only used for the Sfdr view page. So it should be named in a way that this is clear.
2.) The type `SfdrKpiObject` should be defined in `SfdrPanel`, since the actual sfdrKpiObjects are created there and are passed down to `CompanyDataTable`.
3.) You should then also use this type inside of the `createKpiDataObjects()` method.

Reply:
Done. Moved and exporting the type from SfdrPanel.
Not sure applying to `createKpiDataObjects()` though. Slightly different.

Reply:
![image](https://github.com/d-fine/Dataland/assets/99255296/cfb34978-f491-4fc0-856b-01c0ffca38f4)

The red marked object is a SfdrKpiObject, isn't it?

If yes, it should also get the type. That's what I meant with point 3

Reply:
Right. Yes. Done.

Reply:
👍